docker/Dockerfile

```
@@ -339,6 +339,16 @@ RUN apt-get update && apt-get install -y \
       libssl-dev \
       libpcre3-dev
 
+# debmake: a native Debian tool for preparing sources for packaging
+RUN apt-get update && apt-get install -y \
+      debmake \
+      python3-debian
+
+# Packages for jool
+RUN apt-get update && apt-get install -y \
+      libnl-3-dev \
```

Without this, it fails to build for me
```suggestion
      libnl-genl-3-dev \
```

---

Thanks for your attentiveness! I copied the wrong package from the list.
Fixed.